apispec/core.py

```diff
@@ -387,3 +393,26 @@ def validate_openapi_version(openapi_version_str):
         )
 
     return openapi_version
+
+
+class YAMLDumper(yaml.Dumper):
+    pass
+
+    @staticmethod
+    def _represent_dict(dumper, instance):
+        return dumper.represent_mapping('tag:yaml.org,2002:map',
+                                        instance.items())
+
+    if PY2:
+        def _unicode_representer(dumper, uni):
```

Comment:
This should probably be `_represent_unicode` for consistency.

Reply:
Done.